yarn.lock

```diff
@@ -5269,6 +5808,17 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@types/react@npm:>=16, @types/react@npm:^18.2.21":
```

Comment:
Seems this is causing issues. Would a `dedupe` help?

Reply:
I ran yarn dedupe but it doesn't seem to fix CI failures.

I don't really know what's the problem here 😅 any idea?

Reply:
Tried to fix it, seems to work but dedupe actually creates problems 😅